.gitignore

```diff
@@ -30,5 +30,10 @@ target
 # Environment
 /.env
 
+# Generated code
+/**/gen/*
+/**/openapi/*
+!/**/openapi/fs.go
+
```

Comment:
Generovany kod:
- komplikuje review
- robi konflitky pri rebase, ak sa zmenil design

To ze generovany kod je OK budu validovat testy, nepotrebujeme ho mat v Git repe.